dataactbroker/fileRoutes.py

```diff
@@ -384,11 +388,13 @@ def delete_submission(submission):
                  'reporting_fiscal_year': webargs_fields.String(required=True),
                  'reporting_fiscal_period': webargs_fields.String(required=True)})
     def check_year_and_quarter(cgac_code, frec_code, reporting_fiscal_year, reporting_fiscal_period):
-        """ Check if cgac code, year, and quarter already has a published submission """
-        sess = GlobalDB.db().session
-
-        return find_existing_submissions_in_period(sess, cgac_code, frec_code, reporting_fiscal_year,
-                                                   reporting_fiscal_period)
+        """ Check if cgac (or frec) code, year, and quarter already has a published submission """
+        if cgac_code or frec_code:
```

Comment:
If you're returning an error you can check if not cgac_code and not frec_code and then return. Then you don't need an else. This isn't necessarily incorrect but we want to stay consistent.